dashboard/modules/job/job_manager.py

```diff
@@ -206,13 +206,20 @@ async def run(
                 "runtime_env": self._runtime_env,
                 "metadata": self._metadata,
             })
-            # Set RAY_ADDRESS to local Ray address, if it is not set.
+            # Always set RAY_ADDRESS as redis address for job submission.
+            # In case of local development, prevent user from re-using
+            # http://{address}:{dashboard_port} to interact with jobs SDK.
             os.environ[
                 ray_constants.RAY_ADDRESS_ENVIRONMENT_VARIABLE] = \
```

Comment:
previously this could be "http://127.0.0.1:8265" which didn't work, or match what we want to enforce as redis address of current ray cluster.

Reply:
The reason the user tried this was because of the following part of the docs:

> If we have RAY_ADDRESS environment variable set with a local Ray cluster, or just manually set it first:
> ```
> export RAY_ADDRESS="http://127.0.0.1:8265"
> ```


Is this still accurate?  I'm not quite sure what this part of the doc is saying (does the "or" mean there's two recommended options, or is one better than the other?,). Maybe we can clarify it in this PR or a followup PR.

Reply:
Ah @architkulkarni that's correct user behavior, but the issue here is the job manager internally calls `ray.init`, and for that one we want the address to be the address of redis on the local node.